### DISCUSSION TITLE
Some characters escaped the filter!

### DISCUSSION THREAD

**Discussion by haikalnashuha** [tyxla/remove-accents#40] [General]
Hello. First of all, thank you alot for this package! I have been using it and it has been a great help for me.

However yesterday some characters slipped away from the filter. I don't know what you call this character as but here is the code that I replicated from the system. 
```
import accents from "remove-accents";

const oriString = 'Thanh Trì, Hoàng Mai, Hà Nội, 514 Nguyễn Khoái, Hai Bà Trưng, Vietnam';

console.log('ORIGINAL', oriString);
console.log('FILTERED', accents.remove(oriString));
```

and here is the result:
```
ORIGINAL Thanh Trì, Hoàng Mai, Hà Nội, 514 Nguyễn Khoái, Hai Bà Trưng, Vietnam
FILTERED Thanh Tri, Hoang Mai, Ha Nội, 514 Nguyễn Khoai, Hai Ba Trung, Vietnam
```

Maybe you can include the ộ and ễ characters in the filter for future updates. I really like this package, just wanna help.

Thanks alot again!

**Comment by tyxla** [comment]
Thanks for opening the discussion up, @haikalnashuha!

It appears that this is going to be resolved by #43, which is almost ready to be landed.

**Comment by tyxla** [comment]
I'm going to close this one as completed but feel free to contribute to #43 or an alternative solution in a PR. Thanks again!

**Comment by tyxla** [comment]
[Version 0.5.0 has been released](https://github.com/tyxla/remove-accents/releases/tag/v0.5.0) and it contains support for all Vietnamese diacritics as added in #46.
